### DISCUSSION TITLE
这个插件对OpenWrt版本啥的有没有要求？

### DISCUSSION THREAD

**Discussion by drw2005** [UIforFreedom/UIF#24] [General]
有没有最低版本和依赖啥的要求？准备在软路由上安装试试。

**Comment by UIforFreedom** [comment]
文档里面有写，建议使用 docker 方式安装

**Reply by drw2005** [reply]
一键脚本不行吗？

**Reply by UIforFreedom** [reply]
> 一键脚本不行吗？

可以啊，如果你能自行处理依赖问题的话。如果你不懂如何处理，直接用 docker 即可

**Reply by drw2005** [reply]
所需依赖没看到文档里说明啊，能说一下不？

**Reply by UIforFreedom** [reply]
> 所需依赖没看到文档里说明啊，能说一下不？

可以啊，如果你能请我喝杯星巴克的话，我手把手教你。我的 telegram：https://t.me/ui4freedom
